vault/provider.go

```diff
@@ -157,6 +158,25 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("VAULT_NAMESPACE", ""),
 				Description: "The namespace to use. Available only for Vault Enterprise",
 			},
+			"headers": {
```

Comment:
Should this field be marked [sensitive](https://www.terraform.io/docs/cloud/workspaces/variables.html#sensitive-values)? I ask because I suspect that sometimes people will send authentication headers through.

Reply:
Yeah, thats a good idea, I'll change that.